src/Glpi/Application/ErrorHandler.php

```diff
@@ -499,7 +499,7 @@ private function outputDebugMessage(string $error_type, string $message, string
                     $verbosity = OutputInterface::VERBOSITY_VERY_VERBOSE;
                     break;
             }
-            $message = $error_type . ': ' . $message;
+            $message = $error_type . ': ' . \htmlspecialchars($message);
```

Comment:
This case corresponds to a console output. The console is not subject to XSS.

Escaping should be done on the 508 and 509 lines.

Reply:
Fixed 👌